conans/test/unittests/client/build/cmake_test.py

```diff
@@ -202,7 +202,7 @@ def cmake_generator_intel_test(self):
         conanfile.settings = settings
 
         cmake = CMake(conanfile)
-        self.assertIn('-G "Visual Studio 15 2017" -A "x64"', cmake.command_line)
+        self.assertIn('-G "Visual Studio 15 2017 Win64"', cmake.command_line)
```

Comment:
This seemed a bug, that it was using the Intel compiler version (19), instead of the actual CMake generator version